custom-connectors/PUGGamifiedEngagement/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "INTEGRATION API KEY",
+          "description": "INTEGRATION API KEY para esta API.",
+          "tooltip": "Provide your INTEGRATION API KEY.",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```

Comment:
@smart-jp Please make sure that `iconBrandColor` value is the background color of your connector's icon.